py3status/modules/diskdata.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+"""
+Display advanced disk usage information
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds.
+        (default 10)
+    disk: disk whose stat to check. Set to 'all' to get global stats.
+        (default None)
+    format: format of the output.
+        (default "{disk}: {used_percent}% ({total})")
+    format_rate: format for the rates value
+        (default "{value:5.1f} {unit:>4s}")
+    format_space: format for the disk space values
+        (default "{value:3.1f}")
+    sector_size: size of the disk's sectors.
+        (default 512)
+    thresholds: thresholds to use for color changes
+        *(default {'free': [(0, 'bad'), (10, 'degraded'), (100, 'good')],
+        'total': [(0, "good"), (1024, 'degraded'), (1024*1024, 'bad')]})*
+
+Format placeholders:
+    {disk} the selected disk
+    {free} free space on disk in GB
+    {used} used space on disk in GB
+    {used_percent} used space on disk in %
+    {read} reading rate
+    {total} total IO rate
+    {write} writing rate
+
+Rate format placeholders:
+    {unit} name of the unit
+    {value} numeric value of the rate
```

Comment:
Space format has no love it should have things defined too

maybe `format_rate placeholders:`